source/kraken/tests/disruption_reader_test.cpp

```diff
@@ -599,3 +634,57 @@ BOOST_AUTO_TEST_CASE(two_ptobjects) {
     ptobject = impact.informed_entities(1);
     BOOST_CHECK_EQUAL(ptobject.uri(), "uri2");
 }
+
+BOOST_AUTO_TEST_CASE(two_properties) {
+    navitia::type::PT_Data pt_data;
+    navitia::type::MetaData meta;
+    navitia::DisruptionDatabaseReader reader(pt_data, meta);
+
+    Const_it const_it;
+    const_it.set_disruption("1", "22", "33", "44", "55",
+            "note", "reference");
+    const_it.set_cause("1", "wording", "11", "22");
+    const_it.set_tag("1", "name", "11", "22");
+    const_it.set_impact("1", "11", "22");
+    const_it.set_severity("2", "wording", "22", "33",
+            "blocking", "color", "2");
+    const_it.set_application_period("0", "1", "2");
+    const_it.set_ptobject("id", "uri", "line", "1", "2");
+    const_it.set_property("foo", "bar", "31");
+    reader(const_it);
+    const_it.set_property("key", "type", "42");
```

Comment:
Using a concatenation of the 3 fields as key doesn't work:

```c++
const_it.set_property("keyty", "pe4", "2");// KABOOM
```

Reply:
hmmm, I'm sad I didn't see this earlier. I have to use the property ID I think. If I do this, I'll have to make some changes in chaos-proto then.

Reply:
Can't you just put the object directly in the disruption?

Reply:
What about the uniqueness check then ? If the SQL request changes and makes properties appear multiple times isn't it risky ?

Reply:
you can have something like a `std::set<Property>` in the `Disruption`.

Reply:
I rewrited everything without using holder and I replaced the Disruption properties with a set. Tell me if it's ok !